lib/commands/element.js

```diff
@@ -40,6 +40,26 @@ commands.getText = async function (elementId) {
   return await this.uiautomator2.jwproxy.command(`/element/${elementId}/text`, 'GET', {});
 };
 
+commands.click = async function (elementId) {
+  return await this.uiautomator2.jwproxy.command(`/element/${elementId}/click`, 'POST', {elementId});
+};
+
+commands.tap = async function (elementId, x = 0, y = 0, count = 1) {
+  for (let i = 0; i < count; i++) {
+    if (elementId) {
+      // we are either tapping on the default location of the element
+      // or an offset from the top left corner
+      if (x !== 0 || y !== 0) {
```

Comment:
I think the repetition of the actual command is bad. It would be better to get the correct parameters, and then send it.

```js
let params = {elementId};
if (x !== 0 || y !== 0) {
  params[x] = x;
  params[y] = y;
}
await this.uiautomator2.jwproxy.command(`/element/${elementId}/click`, 'POST', params);
```

This could be extended to include the other call, as well.